utils/cloud_cleanup/azure/clean_azure.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2024 ScyllaDB
+
+import argparse
+import os
+from datetime import datetime, timedelta
+
+from sdcm.utils.azure_utils import AzureService
+
+azure_service = AzureService()
+resource_client = azure_service.resource
+compute_client = azure_service.compute
+
+
+def get_keep_hours(v_m, default=14):
+    keep = v_m.tags.get('keep') if v_m.tags else None
```

Comment:
we need `keep_action`

Reply:
Do we really have something else than the `terminate` keep action?

Reply:
we have definition for putting there `stop`, SCT doesn't use it
and that's the default if not defined, i.e. if someone isn't aware of the rules, it would stop it.